R/summary.R

```diff
@@ -370,19 +351,15 @@ setMethod(
   }
 )
 
-
-
 # Print Table Structure ----
 
-#' Summarize Table
+#' Summarize table
 #'
-#' @param x a table object
-#' @param detail either `row` or `subtable`
+#' @param x (`TableTree`)\cr a table object.
```

Comment:
it should be like VTableTree because it covers both tabletree and elementarytable (this applies also to the other instances)

Reply:
I think it is fine TableTree though, better than ElementaryTable